fastlane/swift/Runner.swift

```diff
@@ -22,36 +22,42 @@ func desc(_: String) {
     // no-op, this is handled in fastlane/lane_list.rb
 }
 
-class AtomicDictionary<Key: Hashable, Value> {
-    private let lockQueue = DispatchQueue(label: "tools.fastlane.serial.queue")
+class UnfairDictionary<Key: Hashable, Value> {
```

Comment:
I'd keep the name as `AtomicDictionary` :) It's more user friendly and describes clearly what this class does. The underlying technology powering the atomic behavior isn't important to users of `AtomicDictionary` IMO

Reply:
Done 🚀